/radar_ns_2:
   radar_locations_decoder_gen5_ros:
      ros__parameters:
        subscriber_topic_robs: /sensors/radar_gen5/radar_manager_gen5/RadarOutputROB2
        subscriber_msg_queue_size: 8

        publisher_topic_locations: /sensors/radar_gen5/radar_locations_decoder_gen5/location_interface
        publisher_locations_to_individual_topics_per_sensor: 1

        desired_freq_locations: 15.15
        freq_tol_locations: 0.1
        publisher_msg_queue_size: 10

        radar_ros_time_converter:
          type: rob_timestamp
          ros_timestamp_correction_in_seconds: 0.0
